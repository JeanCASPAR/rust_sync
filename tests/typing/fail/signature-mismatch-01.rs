use rustre::sync;

sync! {
    #![pass(1)]
    
    node oui(a : int) = (a);

    node non(b : bool) = ()
    where
        o : int = oui(b);
}

fn main() {}
