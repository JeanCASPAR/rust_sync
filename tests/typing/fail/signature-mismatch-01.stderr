error: type error: this argument should have type `int`, but it has type `bool`
 --> tests/typing/fail/signature-mismatch-01.rs:8:23
  |
8 |         o : int = oui(b);
  |                       ^
